Src/WitsmlExplorer.Frontend/components/ContextMenus/LogObjectContextMenu.tsx

```diff
@@ -117,13 +120,32 @@ const LogObjectContextMenu = (
     const targetReference: ObjectReference = toObjectReference(
       checkedObjects[0]
     );
-    const copyJob: CopyComponentsJob = {
-      source: logCurvesReference,
-      target: targetReference,
-      startIndex: logCurvesReference.startIndex,
-      endIndex: logCurvesReference.endIndex
-    };
-    JobService.orderJob(JobType.CopyLogData, copyJob);
+
+    if (
+      logCurvesReference.serverUrl?.toLowerCase() ===
```

Comment:
Is there a reason to not get the modal if we are pasting on the same server?

Reply:
Good question, is this behaviour wanted for copying on the same server?

Reply:
It would make the "paste" behavior consistent, so I think it makes sense.